packages/gatsby-recipes/src/providers/npm/fixtures/package-json/package.json

```diff
@@ -1,4 +1,4 @@
 {
-  "name": "test",
+  "name": "test-npm-provider",
```

Comment:
Jest was complaining about this being the same name as another fixture package.json